### PR TITLE
chore: simplify copy

### PR DESCRIPTION
**Pull request by rolznz**
image

image

image

image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -236,36 +236,39 @@ function App() {
...
+                    href="https://info.getalby.com/voltage"
+                    target="_blank"
+                    className="link"
+                  >
+                    Follow our guide
+                  </a>{" "}
+                  to quickly get a cloud hosted node from Voltage
                 </li>
-                <li>Connect to your Lightning node from the Alby extension</li>
+                <li>Connect the Alby extension to your Lightning node</li>
```
Comment (bumi) [inline src/App.tsx:271]:
```suggestion
                <li>Connect the Alby extension to your LND Lightning node</li>
```

File: src/App.tsx [hunk 2]
```diff
@@ -333,8 +336,9 @@ function App() {
                 </p>
                 {hiddenChannelOptions.length > 0 && (
                   <p className="italic text-xs mb-4 w-full text-center">
-                    Deposit more sats to unlock {hiddenChannelOptions.length}{" "}
-                    more nodes
+                    {hiddenChannelOptions.length} nodes are hidden because they
+                    a higher minimum channel size than your current onchain
+                    balance.
```
Comment (bumi) [inline src/App.tsx:341]: why is that useful information? what am I supposed to do with that info?

Comment (rolznz) [reply src/App.tsx:341]: because if the user only deposits 20,000 sats they would think Alby only has one recommended node. Is that ok?

Comment (rolznz) [reply src/App.tsx:341]: The user should know that they should deposit more sats to have more options.

File: src/App.tsx [hunk 3]
```diff
@@ -333,8 +336,9 @@ function App() {
                 </p>
                 {hiddenChannelOptions.length > 0 && (
                   <p className="italic text-xs mb-4 w-full text-center">
-                    Deposit more sats to unlock {hiddenChannelOptions.length}{" "}
-                    more nodes
+                    {hiddenChannelOptions.length} nodes are hidden because they
```
Comment (MoritzKa) [inline src/App.tsx:339]: they **have** a higher

File: src/App.tsx (hunk 2 above)
Comment (rolznz) [reply src/App.tsx:341]: What about "Increase your onchain balance to see more recommended nodes". CC @MoritzKa

**Pull request merged**